### DISCUSSION TITLE
Map statements and questions

### DISCUSSION THREAD

**Discussion by Stella-xyl** [jspsych/jsPsych#2326] [Q&A] [answered]
Hi, I'm designing an experiment where participants will need to learn a list of trivia statements and then answer some trivia questions. The procedure is as below:
Round 1  List A learning (5 statements)
Round 2 List B test (5 new questions)
Round 3 List A test (5 questions learned in Round 1)
(Note: all questions should be randomized to compose the List A and List B)
My code is:

    /* create a statement list of 10 */
    var statements = [
        "In a website browser address bar, 'www' stands for 'World Wide Web'. <br>",
        "Kip is the unit of currency in Laos. <br>",
        "Luxury vehicles such as Ferrari, Maserati, and Lamborghini are manufactured in Italy. <br>",
        "Vatican City is the smallest country in the world. <br>",
        "Babies were born without kneecaps. <br>"
        "Marie Curie was the first woman to win a Nobel Prize (1903). <br>",
        "Atlantic Ocean is saltier than the Pacific Ocean. <br>",
        "French Fries are originally from Belgium. <br>",
        "The Cold War ended in 1989. <br>",
        '"E" is the most common letter in the English alphabet. <br>'];

    /* create a question list of 10 */
    var questions = [
        "In a website browser address bar, what does 'www' stand for?",
        "What is the unit of currency in Laos?",
        "Luxury vehicles such as Ferrari, Maserati, and Lamborghini are manufactured in what country?",
        "What is the smallest country in the world?",
        "Which bone are babies born without?",
        "Who was the first woman to win a Nobel Prize in 1903?",
        "Which one is saltier, the Atlantic Ocean or the Pacific Ocean?",
        "Where are French Fries originally from?",
        "When did the Cold War end?",
        "What is the most common letter in the English alphabet?"];

    /* create List A containing random 5 statements from the statement list */
    var ListA_statements = jsPsych.randomization.sampleWithoutReplacement(statements, 5).join(" ");

    /*map the 5 statements in List A to their questions*/ ？？？
    //var ListA_questions = 

    /* create List B containing the rest 5 questions from the question list */ 
    // var ListB = questions.filter(function(v){ return ListA_questions.indexOf(v) == -1 });

So my question is how to map the 5 statements selected and displayed in List A to the corresponding 5 questions that will be displayed in the test round? Thank you!!

**Comment by Stella-xyl** [comment, marked as answer]
Just solved it myself! :)

**Reply by jesuscgaliano** [reply]
Could you tell us how? :)
